src/main/java/org/example/c06/arrayList/Album.java

```diff
@@ -1,4 +1,4 @@
-package org.example.c06.ArrayList;
+package org.example.c06.arrayList;
```

Comment:
The package naming convention is to use only lowercase letters. Also, in this case, you should merge the chapter number with the chapter name to avoid extra depth. So correct would be 
`package org.example.c06arraylist;`
This is the way we did also in happycoding project.

Reply:
done